icicle/tests/polynomial_test.cu

```diff
@@ -38,6 +59,16 @@ namespace msm {
     return CONCAT_EXPAND(CURVE, G2MSMCuda)(scalars, points, msm_size, config, out);
   }
 
+  cudaError_t
+  _G2MSM(const scalar_t* scalars, const dummy_g2_t* points, int msm_size, MSMConfig& config, dummy_g2_t* out)
+  {
+    scalar_t* scalars_host = static_cast<scalar_t*>(malloc(msm_size * sizeof(scalar_t)));
```

Comment:
note that we don't release the memory

Reply:
I think I have a habit of releasing CUDA memory by now, but not CPU memory lol